### PR TITLE
News screen

### PR DESCRIPTION
**Pull request by sudo-ulmas**
[//]: # (Feel free to remove parts that do not relate to the changes you are willing to bring into main)

### News Screen
[//]: # (Give a brief summary of why the change is being implemented, i.e. feature was built, bug fixing and etc.)

### Changes
[//]: # (List change details here for Code Reviewers)
1. new create tablewview
2. create table view cell
3. add constraints
4 populate with mock data

[//]: # (Use the table below if you have screenshots/videos to show visual updates)

Screenshot 2023-01-23 at 12 57 10 PM

### CODE REVIEW

File: SteamList/Views/NewsTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .white
+        label.font = .systemFont(ofSize: 16, weight: .light)
+        
+        return label
+    }()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        contentView.addSubview(titleLabel)
```
Comment (AbboskhonShukurullaev) [inline SteamList/Views/NewsTableViewCell.swift]: you could extract all those changes to a method `setupContentView`

**Review by AbboskhonShukurullaev — APPROVED** [review]

File: SteamList/Controllers/NewsViewController.swift [hunk 2]
```diff
@@ -8,9 +8,47 @@
 import UIKit
 
 final class NewsViewController: UIViewController {
-
+    
+    private var news = MockData.news
+    private let tableView = {
```
Comment (abubakrj) [inline SteamList/Controllers/NewsViewController.swift:14]: A new line should be good

File: SteamList/Controllers/NewsViewController.swift [hunk 3]
```diff
@@ -8,9 +8,47 @@
...
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        tableView.frame = view.bounds
     }
+    
+    func configureNavbar() {
+        navigationItem.title = "News"
+        navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Filter", style: .plain, target: self, action: nil)
+        navigationItem.rightBarButtonItem?.tintColor = .white
+
```
Comment (abubakrj) [inline SteamList/Controllers/NewsViewController.swift]: Extra line

File: SteamList/Views/NewsTableViewCell.swift (hunk 1 above)
Comment (abubakrj) [reply SteamList/Views/NewsTableViewCell.swift]: Agree, but `func constructHierarchy()` would sound better

File: SteamList/Controllers/NewsViewController.swift [hunk 4]
```diff
@@ -8,9 +8,47 @@
...
+        view.addSubview(tableView)
+        configureNavbar()
+    }
+    
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        tableView.frame = view.bounds
     }
+    
+    func configureNavbar() {
```
Comment (abubakrj) [inline SteamList/Controllers/NewsViewController.swift]: This method can be private

**Review by abubakrj — APPROVED** [review]
Looks good! I have added some minor comments

**Pull request merged**